Formula/create-api.rb

```diff
@@ -0,0 +1,30 @@
+class CreateApi < Formula
+  desc "Delightful code generator for OpenAPI specs"
+  homepage "https://github.com/CreateAPI/CreateAPI"
+  url "https://github.com/CreateAPI/CreateAPI/archive/refs/tags/0.0.5.tar.gz"
+  sha256 "c250ff140af83e093d86fef0dd18b87363ae91a087c0804bedb40f9922989093"
+  license "MIT"
+  head "https://github.com/CreateAPI/CreateAPI.git", branch: "main"
+
+  depends_on xcode: "13.0"
+
+  uses_from_macos "swift"
+
+  resource "homebrew-test-data" do
+    url "https://raw.githubusercontent.com/CreateAPI/CreateAPI/4a0154ddee452b381b4fcbe0a133cd90d6a904fa/Tests/CreateAPITests/Specs/cookpad.json"
+    sha256 "d74e213c00ece19b506f78431da89b782dd84b13ea2ead3572154d85a3f7ffeb"
+  end
+
+  def install
+    system "swift", "build", "--disable-sandbox", "--configuration", "release"
+    bin.install ".build/release/create-api"
+  end
+
+  test do
+    resource("homebrew-test-data").stage testpath
+    system bin/"create-api", "generate", "--package", "TestAPIKit", "--output", ".", testpath/"cookpad.json"
```

Comment:
Since `cookpad.json` is part of the source tree that we initially had access to, it seems simpler to do something like this:

```suggestion
  def install
    system "swift", "build", "--disable-sandbox", "--configuration", "release"
    bin.install ".build/release/create-api"
    pkgshare.install "Tests/CreateAPITests/Specs/cookpad.json"
  end

  test do
    system bin/"create-api", "generate", "--package", "TestAPIKit", "--output", ".", pkgshare/"cookpad.json"
```

Reply:
Ok yeah I can do that. Does that mean that **cookpad.json** will now be installed with everybody's installation? I'm happy with that if you are? 

I was assuming that the `resources` would only be downloaded when running `brew test create-api` (which most people won't do?) so this is a bit different to before? 

Apologies since I'm not too familiar with the inner workings, this is my first time submitting a formula

Reply:
I might rename the file to test-spec.json as well since the filename might be a little bit confusing out of context.

Reply:
> Ok yeah I can do that. Does that mean that **cookpad.json** will now be installed with everybody's installation? I'm happy with that if you are? 

Yes, that's correct. Installing some additional files for the purposes of testing is a pretty common pattern across formulae. The JSON file likely does not take too much space.

> Apologies since I'm not too familiar with the inner workings, this is my first time submitting a formula

No apologies necessary! You've handled this pull request exceedingly well, especially for a first-timer.

Reply:
> I might rename the file to test-spec.json as well since the filename might be a little bit confusing out of context.

Feel free.

Reply:
All done, thanks for the great feedback @carlocab! 👍